### PR TITLE
R4R:Blocks page develop

### PR DESCRIPTION
**Pull request by wenwu-bianjie**

### CODE REVIEW

File: frontend/src/components/ProposalsDetail.vue [hunk 1]
```diff
@@ -121,7 +121,8 @@
       </div>
       <div class="proposals_detail_table_wrap">
         <spin-component :showLoading="showLoading"/>
-        <blocks-list-table :items="items" :type="'ProposalsDetail'" :showNoData="showNoData" :min-width="tableMinWidth"></blocks-list-table>
+        <!-- <blocks-list-table :items="items" :type="'ProposalsDetail'" :showNoData="showNoData" :min-width="tableMinWidth"></blocks-list-table> -->
```
Comment (zhjibi742626083) [inline frontend/src/components/ProposalsDetail.vue]: delete annotation

File: frontend/src/components/ProposalsDetail.vue [hunk 2]
```diff
@@ -134,7 +135,8 @@
       <div class="proposals_table_title_div" style="margin-top: 0;">Depositors</div>
       <div class="proposals_detail_table_wrap">
         <spin-component :showLoading="showLoading"/>
-        <blocks-list-table :items="depositorItems" :type="'ProposalsDetail'" :showNoData="depositorShowNoData" :min-width="tableMinWidth"></blocks-list-table>
+        <!-- <blocks-list-table :items="depositorItems" :type="'ProposalsDetail'" :showNoData="depositorShowNoData" :min-width="tableMinWidth"></blocks-list-table> -->
```
Comment (zhjibi742626083) [inline frontend/src/components/ProposalsDetail.vue]: delete annotation

File: frontend/src/components/ProposalsDetail.vue (hunk 1 above)
Comment (ww415226370) [reply frontend/src/components/ProposalsDetail.vue]: deleted this place

File: frontend/src/components/ProposalsDetail.vue (hunk 2 above)
Comment (ww415226370) [reply frontend/src/components/ProposalsDetail.vue]: deleted this place

**Pull request merged**